docs/UserGuide.md

```diff
@@ -500,34 +522,43 @@ To sort deliveries by cost in ascending order:
 
 ### Upcoming deliveries: `upcoming`
 
-The `upcoming` command is used to view pending deliveries in VendorVault.
-You can choose to view all pending deliveries within a specified date range or
+View pending deliveries in VendorVault. You can choose to view all pending deliveries within a specified date range or
 before or after a given date.
 
-Format: `upcoming aft/START_DATE bef/END_DATE`
+Format: `upcoming aft/DELIVERY_DATE_TIME bef/DELIVERY_DATE_TIME`
+
+<box type="details" seamless>
 
 Parameters:
 
-- `aft/START_DATE`: Must be in dd-mm-yyyy hh:mm format.
-- `bef/END_DATE`: Must be in dd-mm-yyyy hh:mm format.
-<box type="tip" seamless>
-**Tip:** You can provide both START_DATE and END_DATE!
+- `aft/DELIVERY_DATE_TIME`: `DELIVERY_DATE_TIME` is the start date and time in which only deliveries with status`PENDING` after this date and time would be displayed. It must be in dd-mm-yyyy hh:mm format.
+- `bef/DELIVERY_DATE_TIME`: `DELIVERY_DATE_TIME` is the end date and time in which deliveries with status `PENDING` before this date and time would be displayed. It must be in dd-mm-yyyy hh:mm format.
+
+**Tip:**
+- You can provide both parameters!
 
-**Warnings**:
-- A spacing between `upcoming` and the first parameter is compulsory
-- At least one parameter must be provided
-- No duplicate prefix can be used
-- The prefixes `aft/` and `bef/` are **case-sensitive**
 </box>
 
-#### Example
+<box type="warning" seamless>
+
+**Warnings**:
+- At least one space is needed between `upcoming` and the first parameter.
+- At least one parameter must be provided.
+- If both parameters are provided, then only deliveries with status`PENDING` and `DELIVERY_DATE_TIME` between the two specified parameters are displayed (not inclusive)
```

Comment:
Perhaps the not inclusive can be bolded to highlight it since most of the 'between' used in the UG are inclusive?